docs/support.mdx

```diff
@@ -11,7 +11,9 @@ import MainCard from "/src/components/MainCard";
 
 We encourage you to take a look at our [extensive documentation](/) to find quick solutions and assistance.
 
-Didn’t find what you were looking for? We’ve got you covered!
+## Didn’t find what you were looking for? 
```

Comment:
Lets's keep this as just text, not an h2 header since having it has an h2 doesn't line up with the page structure